### ISSUE TITLE
To add: Why We Roll landing page

### ISSUE THREAD

**Issue by Wythemars** [stillfleet/stillfleet.github.io#20]
- [x] Create Why We Roll page (top of site) (#29)
- [x] Add image to be provided by Chris
- [ ] Add menu "Why We Roll" under "About"

Content: I wrote some copy for the WWR page and sent it to Chris to review. I'll ping you when content is locked.

Menu: We agreed it would live underneath "About" as a new, solo submenu item called "Why We Roll." Over time, if we want to move it/add pages, we can discuss.

Question: Can we add an RSS feed thing that shows the episodes in a neat little table? I feel like I've seen this around. I can pull examples, but you may know what I mean, or this may be a headache. Not a big deal either way!

**Comment by armahillo** [comment]
> Question: Can we add an RSS feed thing that shows the episodes in a neat little table? I feel like I've seen this around. I can pull examples, but you may know what I mean, or this may be a headache. Not a big deal either way! - @Wythemars 

Do you want like, an _actual_ RSS feed (consumable by RSS readers) or are you wanting a list of episodes? 

Let me know when you're ready to roll with this! (no pun intended)

**Comment by Wythemars** [comment]
Okay, Chris and I reviewed copy, and we are good here! Please add the page, sans auto-RSS-fed table of episodes for now. LMK if you have any questions/blocks here!

**Comment by armahillo** [comment]
@Wythemars Waiting on image to be provided by Chris

**Comment by armahillo** [comment]
Also - do you want the feature on the homepage to connect with this page somehow?

**Comment by Wythemars** [comment]
Naw, for now let's just get the page up. I'll ping Chris again...

**Comment by armahillo** [comment]
ok you got it!

**Comment by Wythemars** [comment]
![WWR_Site_150dpi](https://github.com/stillfleet/stillfleet.github.io/assets/119684043/68fe1dde-e4e9-41f8-8af8-bd08367c5249)

**Comment by Wythemars** [comment]
Chris made an image. LMK if this doesn't work!

**Comment by armahillo** [comment]
This is great, but I'm not sure it will work in the right-side because I'm concerned the right side will get cut off. Let me see how it works!

**Comment by Wythemars** [comment]
Okay, let me know. We figured that since all of them are letter in aspect
ratio, this vertical rectangle would work. We can make something truly
letter aspect if needed.

Best,

Wythe

<https://twitter.com/hollowearths>Twitter <https://twitter.com/hollowearths>
Podcast: FIELDS <https://heritageradionetwork.org/series/fields>

**Comment by armahillo** [comment]
It's not the aspect ratio that I'm unsure on -- it's that the image has horizontal symmetry (both your faces) which works a little better visually when it's stacked in a vertical webflow - like this would be 100% perfect if it was within content that was in single column. In a two column layout, this also has two columns of its own. It's some nuanced nitpicking, but just weird gut-feels I have after doing this a long time. 

I was also concerned that your face would get cropped off because previously the covers were bleeding off the page, but I think we changed that, so it *shouldn't* be an issue. 

Let me try it out first and I'll get back to you!

**Comment by Wythemars** [comment]
Okay, I mean, let’s try the graphic design as Chris has composed it and then reassess? Chris is quite good at graphic design, and they’ve reviewed our site, so I’m trusting it’ll look fine.Sent via auxiliary rhizosphereOn Feb 28, 2024, at 10:43 AM, Aaron H ***@***.***> wrote:﻿
It's not the aspect ratio that I'm unsure on -- it's that the image has horizontal symmetry (both your faces) which works a little better visually when it's stacked in a vertical webflow - like this would be 100% perfect if it was within content that was in single column. In a two column layout, this also has two columns of its own. It's some nuanced nitpicking, but just weird gut-feels I have after doing this a long time.
I was also concerned that your face would get cropped off because previously the covers were bleeding off the page, but I think we changed that, so it shouldn't be an issue.
Let me try it out first and I'll get back to you!

**Comment by armahillo** [comment]
@Wythemars Grab an example of how you want the episode list to look and put it into a new issue :+1:

**Comment by Wythemars** [comment]
Hey Aaron, this is marked as completed, but I don't see the page on our site. Is it simply missing from the menu?

**Comment by Wythemars** [comment]
I guess a lot of these widgets seem to cost money, e.g.:

https://help.spreaker.com/en/articles/3753188-adding-the-embedded-player-to-external-websites-and-blogs
https://podcastpage.io/embedded-podcast/

Best,

Wythe

<https://twitter.com/hollowearths>Twitter <https://twitter.com/hollowearths>
Podcast: FIELDS <https://heritageradionetwork.org/series/fields>

**Comment by armahillo** [comment]
whoops! you did indicate that and i forgot to make a checklist item for it.
ill get that today.

**Comment by armahillo** [comment]
for automatic, yeah. let me see if the apple podcasts player has an RSS
feed proper; if it does incan write something either in JS or manually to
generate it.

**Comment by Wythemars** [comment]
<img width="1401" alt="Screenshot 2024-03-02 at 5 52 23 PM" src="https://github.com/stillfleet/stillfleet.github.io/assets/119684043/b557e061-3cfd-4087-abb9-a14093e70258">

Why are the menu arrows different?

**Comment by armahillo** [comment]
ou weird! idk will look into it

**Comment by armahillo** [comment]
![Screenshot 2024-03-11 at 3 11 52 PM](https://github.com/stillfleet/stillfleet.github.io/assets/502363/018db029-22dd-4edd-873f-239f5a2f9851)

fixed! 

it was an autocomplete error

**Comment by Wythemars** [comment]
Great, thanks!

Best,

Wythe

<https://twitter.com/hollowearths>Twitter <https://twitter.com/hollowearths>
Podcast: FIELDS <https://heritageradionetwork.org/series/fields>
